controller/lib/core/controller.h

```diff
@@ -0,0 +1,44 @@
+#ifndef CONTROLLER_H_
+#define CONTROLLER_H_
+
+#include "actuators.h"
+#include "blower_fsm.h"
+#include "network_protocol.pb.h"
+#include "pid.h"
+#include "units.h"
+
+// This class is here to allow integration of our controller into Modelica
+// software and run closed-loop tests in a simulated physical environment
+class Controller {
+public:
+  Controller();
+
+  ActuatorsState Run(Time now, const VentParams &params,
+                     const SensorReadings &readings);
+
+private:
+  float ComputeFanPower(Time now, const BlowerSystemState &desired_state,
```

Comment:
Nit, could we have a comment indicating that the fan power ranges in [0-1]?  (Should have been in an earlier PR.)

Reply:
Oops, somehow lost the comment in the move. Restored.